runtime/syntax/ruby.yaml

```diff
@@ -1,26 +1,68 @@
 filetype: ruby
 
 detect: 
-    filename: "\\.rb$|\\.gemspec$|Gemfile|config.ru|Rakefile|Capfile|Vagrantfile"
+    filename: "\\.rb$|\\.gemspec$|^(Gemfile|config.ru|Rakefile|Capfile|Vagrantfile)$"
```

Comment:
What about also adding filenames that end in `.rake`? These are currently uncaught by anything.